src/pages/OrderMenu/OrderMenu.module.css

```
@@ -42,21 +42,27 @@ main {
-  background-color: #ececec;
+  background-color: #ffffff;
 }
 
 .menuCategory {
-  box-sizing: border-box;
-  width: 100%;
-  display: flex;
-  flex-direction: column;
+  border-bottom: 12px solid #ececec;
+  padding-bottom: 2px;
   padding: 0 16px;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)